system/containerd/containerd.SlackBuild

```diff
@@ -26,7 +26,7 @@
 cd $(dirname $0) ; CWD=$(pwd)
 
 PRGNAM=containerd
-VERSION=${VERSION:-1.7.14}
+VERSION=${VERSION:-1.7.20}
 GITHASH=${GITHASH:-dcf2847247e18caba8dce86522029642f60fe96b}
```

Comment:
Can you bump the git hash (8fc6bcff51318944179630522a095cc9dbf9f353) too?

Reply:
done